tests/integration/debug/components.py

```diff
@@ -0,0 +1,95 @@
+import io
+from typing import Tuple, Dict, Union
+
+import numpy as np
+
+from jina.executors.crafters import BaseCrafter
+from jina.executors.decorators import single
+from .helper import _crop_image, _move_channel_axis, _load_image
+
+
+class ImageReader(BaseCrafter):
+    """
+    Load image file and craft it into image matrix.
+
+    :class:`ImageReader` loads the image from the given file
+        path and save the `ndarray` of the image in the Document.
+
+    :param channel_axis: the axis id of the color channel.
+        The ``-1`` indicates the color channel info at the last axis
+    """
+
+    def __init__(self, channel_axis: int = -1, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.channel_axis = channel_axis
+
+    @single(slice_nargs=2)
+    def craft(self, buffer: bytes, uri: str, *args, **kwargs) -> Dict:
+        """
+        Read image file and craft it into image matrix.
+
+        Read the image from the given file path that specified in `buffer` and save the `ndarray` of the image in
```

Comment:
Super small typo: "That **is** specified"

Reply:
Also copy-pasted from the hub just for this test